api/data_refinery_api/views.py

```diff
@@ -540,16 +557,20 @@ class CreateApiTokenView(generics.CreateAPIView):
     """
     token_create
 
-    There're several endpoints like [/dataset](#tag/dataset) and [/results](#tag/results) that return
-    S3 urls where users can download the files we produce, however in order to get those files people
-    need to accept our terms of use by creating a token and activating it.
-
-    ```
-    POST /token
-    PUT /token/{token-id} is_active=True
+    This endpoint can be used to create and activate tokens. These tokens can be used
+    in requests that provide urls to download computed files. They are a way to accept
+    our terms of service.
+
+    ```py
+    import requests
+    import json
+    ​
+    response = requests.post('https://api.refine.bio/v1/token/')
+    token_id = response.json()['id']
+    response = requests.put('https://api.refine.bio/v1/token/' + token_id + '/', json.dumps({'is_activated': True}), headers={'Content-Type': 'application/json'})
     ```
 
-    The token id needs to be sent on the `API_KEY` header on http requests.
+    The token id needs to be attached to the header of the requests with the key `API-KEY`
```

Comment:
I would phrase this `The token id needs to be provided in the HTTP request in the `API-KEY` header.` Just a suggestion though, that might not be as clear to others as it seems to me.

We should definitely have a period at the end of the sentence though :)